catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/MetacardFactory.java

```diff
@@ -118,11 +115,6 @@ Metacard generateMetacard(String mimeTypeRaw, String id, String fileName, Subjec
             generatedMetacard.setAttribute(new AttributeImpl(Metacard.TITLE, fileName));
         }
 
-        String name = Optional.ofNullable(SubjectUtils.getName(subject))
-                .orElse("");
-
-        generatedMetacard.setAttribute(new AttributeImpl(Metacard.POINT_OF_CONTACT, name));
```

Comment:
We do need the point of contact. That was working at some point in the past. It looks like this is actually a bug that we need to fix if this is ending up as null, rather than removing all of this code.